packages/blog-starter-kit/themes/enterprise/components/analytics.tsx

```diff
@@ -178,14 +168,14 @@ export const Analytics = () => {
 		let hasSentBeacon = false;
 		try {
 			if (navigator.sendBeacon) {
-				hasSentBeacon = navigator.sendBeacon(`${BASE_PATH}/api/analytics`, blob);
+				hasSentBeacon = navigator.sendBeacon(`/api/analytics`, blob);
```

Comment:
We don't need the `BASE_PATH` here since the proxy is created on the same domain.